services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/notifications/_notifier.py

```diff
@@ -34,7 +33,7 @@ def __init__(self, sio_manager: socketio.AsyncAioPikaManager):
         self._sio_manager = sio_manager
 
     async def notify_service_disk_usage(
-        self, user_id: UserID, node_id: NodeID, usage: dict[Path, DiskUsage]
+        self, user_id: UserID, node_id: NodeID, usage: dict[str, DiskUsage]
```

Comment:
not sure I get why this was changed from a path.
from the dy-sidecar point of view, that it is via EFS or any other means, that is still a path no?

Reply:
This is for the fronted, before we used to send the disk usage per "path", now we put labels like (INPUTS, OUTPUTS, STATES; NODE) instead of providing paths. These can more easily be handled by the FE and they can display more meaningful information.

Reply:
then why is this not an Enum?

Reply:
good catch will replace